cron/verify_event_fields.py

```diff
@@ -23,18 +23,18 @@ def verify_events(options):
         search_query.add_must_not(ExistsMatch(required_field))
 
         # Exclude all events that are mozdef related health and stats
-        search_query.add_must_not(TermMatch('_type', 'mozdefstats'))
-        search_query.add_must_not(TermMatch('_type', 'mozdefhealth'))
+        search_query.add_must_not(TermMatch('category', 'stats'))
```

Comment:
I wonder if we should name these 'stats' or 'mozdefstats' to be more specific, thoughts?